canonical-web-and-design/ubuntu.com
# Microcerts fix envs

## Done

Fix issues between environments mentioned in https://github.com/canonical-web-and-design/ubuntu.com/pull/10743#issuecomment-957316184

## QA

- I believe best way to QA this is to use [Morgan's PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/10743#issuecomment-957316184)
- Purchase a course and make sure that the buttons change to "Take" now
- Fail a course and check that it's not accessible anymore
- Header should activate the correct course you are taking (no changes from ubuntu.com)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/366
